erpnext/hr/report/daily_work_summary_replies/daily_work_summary_replies.py

```diff
@@ -41,13 +41,20 @@ def get_data(filters):
 		fields=["name"],
 		filters=[["creation","Between", filters.range]])
 	daily_summary_emails = [d.get('name') for d in daily_summary_emails]
-	replies = frappe.get_all('Communication',
-			fields=['content', 'text_content', 'sender'],
-			filters=[['reference_doctype','=', 'Daily Work Summary'],
-				['reference_name', 'in', daily_summary_emails],
-				['communication_type', '=', 'Communication'],
-				['sent_or_received', '=', 'Received']],
-			order_by='creation asc')
+
+	replies = frappe.db.sql("""
+			select `tabCommunication`.content, `tabCommunication`.text_content, `tabCommunication`.sender
+			from `tabCommunication`
+			inner join `tabDynamic Link`
+			on `tabCommunication`.name=`tabDynamic Link`.parent where
+			`tabDynamic Link`.link_doctype in %(daily_summary_emails)s and
```

Comment:
`link_doctype = 'Daily Work Summary' and link_name in %(daily_summary_emails)s`

Reply:
fixed